oeps/oep-0051-bp-conventional-commits.rst

```diff
@@ -0,0 +1,184 @@
+==============================
+OEP-0051: Conventional Commits
+==============================
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-0051 </oeps/oep-0051-bp-conventional-commits>`
+   * - Title
+     - Conventional Commits
+   * - Last Modified
+     - 2021-02-24
+   * - Authors
+     - Ned Batchelder
+   * - Arbiter
+     - Feanil Patel
+   * - Status
+     - Accepted
+   * - Type
+     - Best Practice
+   * - Created
+     - 2021-02-01
+   * - Review Period
+     - Until 2021-03-15
+   * - Resolution
+     - TK
+   * - References
+     - `Conventional Commits`_
+     - `Change Transparency`_
+
+
+Abstract
+========
+
+Commits should be clearly labeled for their effect on consumers of the repository.  To do this, we adopt the `Conventional Commits`_ guidelines, which detail structured commit messages that clarify the impact of each commit.
+
+This is part of our `Change Transparency`_ initiative.
+
+Motivation
+==========
+
+Developers should describe the changes they make.  This information is valuable to many audiences: deployers, integrators, end-users, operators, and so on. Developers are in the best position to accurately describe their work and its implications.
+
+The nearer to the change that information is captured, the easier it will be to write, and the more accurate it will be.  Changes made by developers are consumed by many people, from the pull request reviewer to the final user of the product.  The more the developer can explain the change, the more easily the change will be understood and used, and the more efficient the overall development process will be.
+
+Commit messages are the closest text to the change itself.  The more
+information-rich we can make commit messages, the more useful they will be.
+
+
+Specification
+============
+
+We are adopting the `Conventional Commits`_ spec, with our own adjustments.
```

Comment:
Ideally, we would explicitly call-out the areas where we have diverged from the spec(s).

After reading both, it's not obvious to me (or someone coming into this project who already has experience w/ the original spec and/or Angular), what the immediate differences are.

Reply:
I will take a stab at summarizing the differences.